### DISCUSSION TITLE
[vorta] no selection of a file during a restore

### DISCUSSION THREAD

**Discussion by HDInfautre** [borgbase/vorta#1416] [FAQ] [answered]
Hello

I select the last backup of the day and I want to select a single file to restore it in a directory
Whatever method I use (right click, "mount" button, "extract") I find myself downloading all the files/directories of the backups

I would like to apply simply the step 2a of the documentation
https://vorta.borgbase.com/usage/restore/
obsolete documentation ? or adapted to version 2.0 ? (I am with Borg 1. / vorta 0.8.7 - Ubuntu 22.4)

Regards

**Comment by m3nu** [comment]
Mount and extract are 2 different features. You do one OR the other.

The docs call this 2a and 2b because you choose one of them. 😄

**Reply by HDInfautre** [reply]
Ok sorry for my bad English; I tried both, but what I'm looking for is how to restore a file/directory threshold as an image - step 2a
https://vorta.borgbase.com/usage/restore/#step-2a---extracting-from-an-archive

**Reply by m3nu** [reply]
Yes, that's done using the "Extract" dialog. If there is an issue, please supply the logs from *Misc > Log*, which will have the precise command.

**Comment by HDInfautre** [comment]
Hi
Ok attached file in zip format
Le 07/09/2022 à 13:01, Manu a écrit :

**Reply by real-yfprojects** [reply]
> Ok attached file in zip format

I think that doesn't work via email.

**Reply by HDInfautre** [reply]
Hi
Ok attached file in zip format
==> [vorta.zip](https://github.com/borgbase/vorta/files/9506556/vorta.zip)

**Comment by m3nu** [comment]
There is only one `borg extract` command in those logs and it doesn't have any paths or files selected.

`451:2022-09-05 17:16:31,826 - vorta.borg.borg_job - INFO - Running command /app/bin/borg extract --list --info --log-json ssh://l0m3hxfn@l0m3hxfn.repo.borgbase.com/./repo::lemot-2022-09-05-124349`

On the other hand, if I select a folder, it's added to the command:

`2022-09-07 18:55:33,149 - vorta.borg.borg_job - INFO - Running command /usr/local/bin/borg extract --list --info --log-json /Users/manu/Downloads/test-repo::nyx2.local-2022-08-03-150003 Users/manu/Pictures/portraits`

Maybe you just *selected* the line, but not the *checkbox*? This would be the simplest explanation for your troubles. Looks like this for reference:

<img src="https://user-images.githubusercontent.com/3916435/188936814-54a47600-c0ea-4c02-8f65-006916769386.jpg" width="250" />

To debug further, maybe you want to share a video if explaining it is too difficult?

**Comment by HDInfautre** [comment, marked as answer]
Hi
Ok I get it. I have a black background, I couldn't see the box!
![Capture d’écran du 2022-09-07 19-26-52](https://user-images.githubusercontent.com/6817936/188941819-fc23dfb6-edb5-4c16-abc8-88ebf6a19b6a.png)
